File: API/Entities/Evaluation/Template.cs
using API_MySIRH.Entities.Evaluation;

namespace API_MySIRH.Entities
{
    public class Template:EntityBase
    {
        public string? Technologie { get; set; }
        public string? Them { get; set; }
        public string? Title { get; set; }
        public int? EntretienId { get; set; }
        public Entretien? Entretien { get; set; }

        public int? NotesId { get; set; }
        public Notes? Note { get; set; }
    }
}

File: client/src/app/Models/Template.ts
import { ColumnFreezeContentRenderer } from "@syncfusion/ej2-angular-grids";
import { Base, IBase } from "./Base.model";


        
export interface ITemplate extends IBase {

  technologie:string;
  them:string;
  title :string;
  note :Number;
  EvaluationId:Number;
  CommenterId:number;
}
export class Template extends Base {

      technologie:string;
      them:string;
      title :string;
      EntretienId:Number;
  
}


export class TemplateDTO extends Base{
  NotesId:Number;
}

export class Templatecommente extends Base{
  Commente:string;
}





